src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    return (price -= 50);
+  } else if (days >= 3) {
+    return (price -= 20);
```

Comment:
Similar to the previous comment, you should return 'price - 20' directly here. Also, since the 'if' statement on line 9 already contains a return, you don't need the 'else' keyword. This simplifies the control flow.